libs/cardano-ledger-test/cardano-ledger-test.cabal

```
@@ -126,7 +126,7 @@ library
         mtl,
         nothunks,
         hspec,
-        plutus-ledger-api,
+        plutus-ledger-api ^>=1.18,
```

Omission of bounds here is intentional. We do not release `cardano-ledger-test` to CHaPs. So there is need to put whole bunch of unnecessary bounds in this package
```suggestion
        plutus-ledger-api,
```

---

Fixed.